File: src/test/java/com/mmg/PersonTest.java
package com.mmg;

import org.junit.jupiter.api.Test;
import org.junit.jupiter.api.Assertions;

public class PersonTest {
    @Test
    public void bmiTest() {
        Person person = new Person(66.5f,1.7f);
        Assertions.assertEquals(66.5f/(1.7f*1.7f),person.bmi());
    }

}

File: src/test/kotlin/com/mmg/PersonTest.kt
package com.mmg.kotlin

import com.mmg.Human
import com.mmg.Person
import org.junit.jupiter.api.Assertions
import org.junit.jupiter.api.Test

class PersonTest(){
    @Test
    fun bmiTest(){
        val human = Human("MMG",75f,1.83f)
        Assertions.assertEquals(75f/(1.83f*1.83f),human.bmi())
    }
    @Test
    fun personBmiTest(){
        val person = Person(75f,1.83f)
        Assertions.assertEquals(75f/(1.83f*1.83f),person.bmi())
    }
}